### ISSUE TITLE
Favorites in the Android TV client

### ISSUE THREAD

**Issue by mr-simba** [jellyfin/jellyfin-androidtv#4301]
The Jellyfin Web UI and Android client both have a menu item labeled "Favorites" at the top of the homepage. Where can I find the "Favorites" option in the Android TV client?

**Comment by mr-simba** [comment]
Thank you.

However, the "Favorite" filter does not seem to work with favorite episodes of a TV series or favorite actors, for example, as it does in the "Favorites" menu item in the Android client.

**Comment by nielsvanvelzen** [comment]
Correct, we do not have a dedicated "favorites" page in the app at this moment.

**Comment by nielsvanvelzen** [comment]
Closing as duplicate of #1232
